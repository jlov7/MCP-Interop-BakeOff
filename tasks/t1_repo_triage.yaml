id: t1_repo_triage
name: Repository Triage
description: >
  Inspect core repository files and produce a structured JSON summary that captures
  notable components and potential risks. This task exercises read-only tooling.
runner_prompt: |
  You are assisting with a repository triage. Read the allowlisted files, summarise the
  project purpose, list key services, and note any obvious risks. Return JSON with fields
  {"summary": str, "components": [str], "risks": [str]}.
approval: never
expected_artifact: tasks/artifacts/t1_repo_triage_expected.json
mode: read
