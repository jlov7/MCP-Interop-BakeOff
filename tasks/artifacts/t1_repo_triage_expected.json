{
  "summary": "usb-agents coordinates MCP portability evaluation across three runtimes.",
  "components": [
    "Common MCP server",
    "Deterministic task specs",
    "Runner harness for OpenAI, Microsoft, and Mistral"
  ],
  "risks": [
    "Preview SDK compatibility drift",
    "Telemetry exporter configuration errors"
  ]
}
